css/src/components/popover.css

```diff
@@ -15,3 +15,23 @@
   visibility: hidden;
   pointer-events: none;
 }
+
+@keyframes popper-fade-in {
+  0% {
+    opacity: 0;
+  }
+  80% {
+    opacity: 0.2;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+@keyframes popper-fade-out {
+  0% {
```

Comment:
this same keyframe is present in animation.css with name fadeOut. you may use it